fe/java-udf/src/main/java/org/apache/doris/udf/JdbcExecutor.java

```diff
@@ -1458,4 +1483,56 @@ private int getFirstNotNullObject(Object[] column, int numRows, long nullMapAddr
         }
         return i;
     }
+
+    // only used by nebula-graph
+    public static Object tranferToDorisObject(ValueWrapper value) {
```

Comment:
it‘s better to use  switch case

Reply:
> it‘s better to use switch case and `tranferToDorisObject` is inappropriate

if use switch case we need to know the nebula type , i think here no need to do this since they give the interface to check the type

Reply:
> why not like other database doing convert the work, if you put convert in `getBlock` function, it's will have significant performance impact. eg int type: `copyBatchIntResult`, it convert some object from different database to int, so you need write the else if (..... graph object), then convert to int data insert into column

Graph database is different from relational database , The resultset cannot be directly used, and needs to be converted according to different scenarios, such as using 'vertex' as the varchar type here

As for performance, there may be issues. Currently, it seems that the impact is not significant, and this conversion operation is necessary

Reply:
now all result is save as object, you just check the object instanceof ValueWrapper, and then convert to string.....

Reply:
> now all result is save as object, you just check the object instanceof ValueWrapper, and then convert to string.....

Don't you think there's a lot of redundant code now， Why not unify the result set ？  What's the difference  between 
 convert in getBlock function and convert in copyBatchIntResult ？